spec/classes/foreman_providers_spec.rb

```diff
@@ -8,7 +8,8 @@
 
       context 'with defaults' do
         it { should compile.with_all_deps }
-        it { should contain_package(oauth_os) }
+        it { should contain_package(oauth_os).that_comes_before('Anchor[foreman::providers::oauth]') }
+        it { should contain_anchor('foreman::providers::oauth]') }
```

Comment:
```suggestion
        it { should contain_anchor('foreman::providers::oauth') }
```

Reply:
So I ran the tests and then forgot to rerun them after adding this for resource coverage ... I've made this exact mistake a few times :(